internal/dataplane/kongstate/kongstate.go

```diff
@@ -100,6 +100,17 @@ func (ks *KongState) FillConsumersAndCredentials(log logrus.FieldLogger, s store
 					}
 					continue
 				}
+				// TODO: ttl is a field that only appears in keyAuth credentials and has int type.
```

Comment:
Do we have something to do here? If so let's state it explicitly, otherwise let's drop the `TODO` or change to to `NOTE`. I'd suggest to make it a plain comment, without any "tag" in the beginning though.

Reply:
removed the `TODO`.